rcp/org.testeditor.rcp4.tcltestrun.tests/src/org/testeditor/rcp4/tcltestrun/TclMavenLauncherTest.xtend

```diff
@@ -0,0 +1,48 @@
+package org.testeditor.rcp4.tcltestrun
+
+import java.io.File
+import java.nio.file.Files
+import org.eclipse.core.resources.IProject
+import org.eclipse.core.runtime.IPath
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.runners.MockitoJUnitRunner
+
+import static org.junit.Assert.*
+import static org.mockito.Mockito.*
+
+@RunWith(MockitoJUnitRunner)
+class TclMavenLauncherTest {
+
+	@Mock
+	MavenExecutor executor
+
+	@InjectMocks
+	TclMavenLauncher launcher
+
+	@Test
+	def void testGetProfiles() {
```

Comment:
style: should be reformatted (e.g. `if(...)`) and a space should be entered after `// given`, `// when`, `// then`